pipeline/electron/product-build.yaml

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+jobs:
+    - job: 'build-product-windows'
+
+      pool:
+          vmImage: 'windows-latest'
+
+      steps:
+          - template: ../install-node-prerequisites.yaml
+          - template: electron-e2e-test-interactive.yaml
+          - template: electron-e2e-publish-results.yaml
+
+          - template: distribute-electron.yaml
```

Comment:
Do you think we might need a platform specific distribution pipeline?

Reply:
yes, I think so. This PR is meant to just get something lightweight in for a single platform and we'll have to iterate several times. I'll update the PR description